### ISSUE TITLE
Error deleting time entry

### ISSUE THREAD

**Issue by monday8am** [toggl/mobile#746]
System.InvalidCastExceptionCannot cast from source type to destination type.

```
at Toggl.Joey.UI.Adapters.LogTimeEntriesAdapter.RemoveItemWithUndo (int) <0x00054>
at Toggl.Joey.UI.Fragments.LogTimeEntriesListFragment.OnDismiss (Android.Support.V7.Widget.RecyclerView,int) <0x000e7>
at Toggl.Joey.UI.Utils.SwipeDismissTouchListener.OnItemDismissed (object,System.EventArgs) <0x000f3>
at Toggl.Joey.UI.Views.ListItemSwipeable.OnScrollAnimationEnd (object,System.EventArgs) <0x00067>
at Android.Animation.AnimatorEventDispatcher.OnAnimationEnd (Android.Animation.Animator) <0x0003b>
at Android.Animation.Animator/IAnimatorListenerInvoker.n_OnAnimationEnd_Landroid_animation_Animator_ (intptr,intptr,intptr) <0x0005f>
at (wrapper dynamic-method) object.9db7503e-5371-4181-b84c-6057c607e0c7 (intptr,intptr,intptr) <0x00043>
```

**Comment by monday8am** [comment]
XI: c7a973beacae7fcb6afaf0d133580bd5ec6a3539
still on 8.0.1 version.

**Comment by monday8am** [comment]
Checking in version 8.0.2

**Comment by monday8am** [comment]
Closed in version 8.0.2
